cpu/microarchitectures.json

```
@@ -176,8 +176,47 @@
         ]
       }
     },
-    "nehalem": {
+    "x86_64_v2": {
       "from": ["core2"],
+      "vendor": "generic",
+      "features": [
+        "mmx",
+        "sse",
+        "sse2",
+        "ssse3",
+        "sse4_1",
+        "sse4_2",
+        "popcnt"
+      ],
+      "compilers": {
+        "gcc": [
+          {
+            "versions": "11.1:",
+            "name": "x86-64-v2",
+            "flags": "-march={name} -mtune=generic"
+          },
+          {
+            "versions": "4.6:10.3",
+            "name": "x86-64",
+            "flags": "-march={name} -mtune=generic -mcx16 -msahf -mpopcnt -msse3 -msse4.1 -msse4.2 -mssse3"
+          }
+        ],
+        "clang": [
+          {
+            "versions": "12.0:",
+            "name": "x86-64-v2",
+            "flags": "-march={name} -mtune=generic"
+          },
+          {
+            "versions": "3.9:11.1",
+            "name": "x86-64",
+            "flags": "-march={name} -mtune=generic -mcx16 -msahf -mpopcnt -msse3 -msse4.1 -msse4.2 -mssse3"
+          }
+        ]
+      }
+    },
+    "nehalem": {
+      "from": ["x86_64_v2"],
```

I wonder if:
```suggestion
      "from": ["core2", "x86_64_v2"],
```
is a better modeling choice (and similarly for the other virtual versions below). Need to give some more thought at it.